tensorflow.md

```diff
@@ -1,32 +1,65 @@
-# Building TensorFlow on Graviton2
+# ML inference on Graviton CPUs with TensorFlow
 
-Install bazel build system:
+There are two ways to use tensforflow based ml inferencing on Graviton CPUs.
+- Tensorflow DLAMI
+- Tensorflow Docker container
+
+**Using AWS DLAMI:**
+Launch Graviton instance with the DLAMI: Deep Learning AMI Graviton GPU TensorFlow 2.7.0 (Ubuntu 20.04) 20211122. Now, the platform is ready for start building Tensorflow based ML inference applications. The DLAMI comes with the additional libraries preinstalled, eg: rust compilers, transformers etc, to enable transformers based models (eg: bert) inferencing.
 ```
-# find the latest release of bazel for arm64 https://github.com/bazelbuild/bazel/releases/latest
-# using older bazel versions may not be supported to build tensorflow
-REL=4.2.1  # this is the latest stable release as of this writing
-wget https://github.com/bazelbuild/bazel/releases/download/${REL}/bazel-${REL}-linux-arm64
-chmod +x bazel-${REL}-linux-arm64
-sudo mv bazel-${REL}-linux-arm64 /usr/local/bin
-sudo ln -s /usr/local/bin/bazel-${REL}-linux-arm64 /usr/local/bin/bazel
+# One can check the TF version as below
+pip show tensorflow
+
+# By default eigen backend is enabled, it is recommended to switch to the optimized onednn+acl(arm compute library). This can be enabled by setting the below TF environment variable
+export TF_ENABLE_OENDNN_OPTS=1
+
+# Graviton3 (eg: c7g instances) supports BF16 format for ML acceleration. This can be enabled in oneDNN by setting the below environment variable
+export DNNL_DEFAULT_FPMATH_MODE=BF16
 ```
 
-Build TensorFlow on Graviton2 with Ubuntu 20.04:
+**Using docker container:**
+Launch Graviton instance with Ubuntu20.04 AMI and then setup the docker following the below instructions. The docker image comes with MLPerf inference benchmarks and additional packages required to execute them.
```

Comment:
Minor grammar corrections: "Launch a Graviton instance with an Ubuntu 20.04 AMI and instal docker using the below instructions. The docker image comes pre-installed with the MLPerf inference benchmarks so one can benchmark the install and verify everything is configured properly"